utilities_common/chassis.py

```diff
@@ -16,3 +16,23 @@ def get_chassis_local_interfaces():
                 lst = data[1].split(",")
                 return lst
     return lst
+
+
+def is_smartswitch():
+    return hasattr(device_info, 'is_smartswitch') and device_info.is_smartswitch()
+
+
+def is_dpu():
+    return hasattr(device_info, 'is_dpu') and device_info.is_dpu()
+
+
+def get_num_dpus():
+    if hasattr(device_info, 'get_num_dpus'):
+        return device_info.get_num_dpus()
+    return 0
+
+
+def get_dpu_list():
```

Comment:
@vvolam This may have interactions with the PR https://github.com/sonic-net/sonic-utilities/pull/3271.  Can you please resolve them? Currently do you have any consumer for this?

Reply:
@rameshraghupathy Sure, once PR is merged, I can fix the conflicts. With latest modified commits, I don't have in this current PR.